courses/machine_learning/deepdive/01_bigquery/a_sample_explore_clean.ipynb

```diff
@@ -3630,7 +3630,7 @@
     "To store BigQuery results in a Pandas DataFrame we have have to query the data with a slightly differently syntax.\n",
```

Comment:
<ol><li>Import the&nbsp;<code>google.cloud</code>&nbsp;<code>bigquery</code> module</li></ol><p> </p><p>add backticks around "bigquery" here and below</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/600/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Good call! Added.